requirements.txt

```diff
@@ -1,4 +1,4 @@
-fbs==0.8.6
+fbs
```

Comment:
I think it is still worth to use version pinning here, at least for maj.minor version, so `fbs==1.0`

Reply:
Yes

Reply:
Sorry, I just became aware that for allowing updates on patch level, we have to use the tilde operator (`fbs~=1.0`). With using the `==1.0` operator, it will download 1.0.0, no newer patchlevel like 1.0.1.

Reply:
Yes

Reply:
Changed to `fbs~=1.2`